### PR TITLE
CRISTAL-304: Remove mock element from the early prototype

### PR DESCRIPTION
**Pull request by pjeanjean**
# Jira URL

https://jira.xwiki.org/browse/CRISTAL-304

# Changes

## Description

* The quick page creation button was removed
* Comments and Likes info actions were disabled
* Last edition details were removed (but the span was kept for layout reasons)
* The "Information" extra tab was disabled
* The search input was removed
* The help button now opens the user guide
* The title of the Navigation panel was changed to the name of the current configuration
* The "Applications" panel was removed
* Buttons to open panel actions were removed

## Clarifications

For each change, a new or existing issue was linked to https://jira.xwiki.org/browse/CRISTAL-304 for future improvements.

# Screenshots & Video

image

# Executed Tests

The full test suite was run locally.

# Expected merging strategy

* Prefers squash: Yes 
* Backport on branches:
  * N/A

### CODE REVIEW

File: skin/src/vue/c-sidebar.vue [hunk 1]
```diff
@@ -156,17 +156,14 @@ function onClickOutsideMainSidebar() {
         <u-i-extensions uix-name="sidebar.actions"></u-i-extensions>
       </suspense>
     </div>
-    <div class="search">
-      <x-search></x-search>
-    </div>
     <div class="panel-container">
-      <c-sidebar-panel name="Wiki Name">
+      <!-- TODO: Use wiki name as panel name (CRISTAL-374). -->
+      <c-sidebar-panel :name="cristal.getWikiConfig().name">
```
Comment (manuelleduc) [inline skin/src/vue/c-sidebar.vue:161]: did we agree about this?

File: skin/src/vue/c-help.vue [hunk 2]
```diff
@@ -18,13 +18,27 @@ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
...
+import { useI18n } from "vue-i18n";
+
+const { t } = useI18n({
+  messages,
+});
+
+function onClick() {
+  // TODO: Add an actual Help section (CRISTAL-373).
+  window.open(
+    "https://cristal.xwiki.org/xwiki/bin/view/Documentation/UserGuide/",
```
[the remark this answers is not in the document]
Comment (pjeanjean) [reply skin/src/vue/c-help.vue:32]: I did, and it opened in my Chrome instance (which I documented in https://jira.xwiki.org/browse/CRISTAL-373).

File: skin/src/vue/c-sidebar.vue (hunk 1 above)
Comment (pjeanjean) [reply skin/src/vue/c-sidebar.vue:161]: We did not really discuss it, but it was either this or an empty name. Both are fine to me.

Comment (manuelleduc) [reply skin/src/vue/c-sidebar.vue:161]: I'm +1 for an empty title, but this one might be worth discussing in https://forum.xwiki.org/t/remove-replace-placeholders-in-cristal/16090

File: skin/src/vue/c-help.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (manuelleduc) [reply skin/src/vue/c-help.vue:32]: I'm still +1 to just remove the help button though :)
You should also discuss that one in https://forum.xwiki.org/t/remove-replace-placeholders-in-cristal/16090 to be sure everybody agrees.

**Pull request merged**